app/src/main/admin/components/Users/Create/CreateUserForm.tsx

```diff
@@ -1,55 +1,112 @@
-import { FormConnector, ReformProps } from "alt-reform";
-import { CreateUserFields, createUserFormStore, suggestUsername } from "./CreateUserFormStore";
-import { ValidationError } from "../../../../shared/components/Login/ValidationError";
 import * as React from "react";
+import {change, Field, reduxForm} from "redux-form";
+import {connect} from "react-redux";
+import {compose} from "recompose";
+import {Dispatch} from "redux";
+import {validations} from "../../../../shared/modules/reduxForm";
+import {ReduxFormField} from "../../../../shared/components/ReduxForm/ReduxFormField";
+import {ReduxFormValidationError} from "../../../../shared/components/ReduxForm/ReduxFormValidationError";
+import {AdminAppState} from "../../../reducers/adminAppReducers";
+import {usersActionCreators} from "../../../actions/usersActionCreators";
 
-export class CreateUserFormComponent extends React.Component<ReformProps, undefined> {
-    constructor() {
-        super();
-        this.changeName = this.changeName.bind(this);
+export interface CreateUserFormProps {
+    handleSubmit: (F: Function) => any,
```

Comment:
Remind me again what's the deal with this thing? And maybe add a comment?

Reply:
Seems redux form adds this handler function and wants you to use it on form submission. Not sure what would happen if you just skip this and call our custom `submit` function directly. I might try that actually and see.